src/connector/BlendWithdrawActions.ts

```diff
@@ -25,11 +25,29 @@ export async function withdraw(
   const amount0Min = estimated0.mul(1 - ratioChange / 100);
   const amount1Min = estimated1.mul(1 - ratioChange / 100);
 
+  let transactionOptions: any = {};
+
+  try {
+    const estimatedGas = (
+      (await tokenContract.estimateGas.withdraw(
+        shares.toFixed(0),
+        amount0Min.toFixed(0),
+        amount1Min.toFixed(0)
+      )) as BigNumber
+    ).toNumber();
+
+    transactionOptions['gasLimit'] = (estimatedGas * 1.1).toFixed(0);
+  } catch (e) {
+    console.error('Error while estimating gas');
+    console.error(e);
```

Comment:
Do we have a way of keeping track of this metric? It could be useful to see if this ever occurs and the circumstances in which it occurs if it does occur.

Reply:
We don't have error analytics set up yet, at the moment we're limited to just emitting it in the console.